web_ui/Dockerfile

```diff
@@ -23,9 +23,11 @@ RUN npm run build:rest-openapi-spec
 FROM node:22.12-bookworm-slim@sha256:35531c52ce27b6575d69755c73e65d4468dba93a25644eed56dc12879cae9213 AS web_ui_deps
 
 ARG LOCATION=/home/app/web_ui/
-
 WORKDIR ${LOCATION}
 
+ARG GETI_VERSION=latest
```

Comment:
Please refrain from using the latest tag and instead use the standardised versioning approach that's already applied in the repo. This ensures consistency and proper version tracking across all our components.

Reply:
This should only be used as a fallback when no build argument for the version is given, for example when building locally in which case we don't have access to a specific version.
I can also change it to something like `dev`.